includes/class-wc-gateway-swedbank-pay-checkout.php

```diff
@@ -178,22 +190,24 @@ public function __construct() {
-		$this->auto_capture     = isset( $this->settings['auto_capture'] ) ? $this->settings['auto_capture'] : $this->auto_capture;
-		$this->logo_url         = isset( $this->settings['logo_url'] ) ? $this->settings['logo_url'] : $this->logo_url;
-		$this->instant_checkout = isset( $this->settings['instant_checkout'] ) ? $this->settings['instant_checkout'] : $this->instant_checkout;
-		$this->checkin          = isset( $this->settings['checkin'] ) ? $this->settings['checkin'] : $this->checkin;
-		$this->checkin_country  = isset( $this->settings['checkin_country'] ) ? $this->settings['checkin_country'] : $this->checkin_country;
-		$this->terms_url        = isset( $this->settings['terms_url'] ) ? $this->settings['terms_url'] : get_site_url();
-		$this->save_cc          = 'no';
+		$this->enabled                = isset( $this->settings['enabled'] ) ? $this->settings['enabled'] : 'no';
+		$this->title                  = isset( $this->settings['title'] ) ? $this->settings['title'] : '';
+		$this->description            = isset( $this->settings['description'] ) ? $this->settings['description'] : '';
+		$this->merchant_token         = isset( $this->settings['merchant_token'] ) ? $this->settings['merchant_token'] : $this->merchant_token;
+		$this->payee_id               = isset( $this->settings['payee_id'] ) ? $this->settings['payee_id'] : $this->payee_id;
+		$this->subsite                = isset( $this->settings['subsite'] ) ? $this->settings['subsite'] : $this->subsite;
+		$this->testmode               = isset( $this->settings['testmode'] ) ? $this->settings['testmode'] : $this->testmode;
+		$this->debug                  = isset( $this->settings['debug'] ) ? $this->settings['debug'] : $this->debug;
+		$this->culture                = isset( $this->settings['culture'] ) ? $this->settings['culture'] : $this->culture;
+		$this->auto_capture           = isset( $this->settings['auto_capture'] ) ? $this->settings['auto_capture'] : $this->auto_capture;
+		$this->logo_url               = isset( $this->settings['logo_url'] ) ? $this->settings['logo_url'] : $this->logo_url;
+		$this->instant_checkout       = isset( $this->settings['instant_checkout'] ) ? $this->settings['instant_checkout'] : $this->instant_checkout;
+		$this->checkin                = isset( $this->settings['checkin'] ) ? $this->settings['checkin'] : $this->checkin;
+		$this->checkin_country        = isset( $this->settings['checkin_country'] ) ? $this->settings['checkin_country'] : $this->checkin_country;
+		$this->terms_url              = isset( $this->settings['terms_url'] ) ? $this->settings['terms_url'] : get_site_url();
+		$this->save_cc                = 'no';
+		$this->use_payer_info         = isset( $this->settings['use_payer_info'] ) ? $this->settings['use_payer_info'] : $this->use_payer_info;
+		$this->use_cardholder_info    = isset( $this->settings['use_cardholder_info'] ) ? $this->settings['use_cardholder_info'] : $this->use_cardholder_info;
```

Comment:
The `cardHolder` information is always optional and should only be sent if it is available. It doesn't make sense requiring this information, I think.